src/connection/mod.rs

```diff
@@ -48,40 +48,26 @@ fn test_list_function() {
 }
 
 /// Creates a Network Manager connection.
-///
-/// # Examples
-///
-/// ```
-/// let connection = network_manager::connection::create(
-///     "resin_io",
-///     network_manager::device::DeviceType::WiFi,
-///     network_manager::wifi::Security::WPA2,
-///     "super_secret_passphase"
-///     ).unwrap();
-/// println!("{:?}", connection);
-/// ```
-pub fn create(s: &str, dt: DeviceType, sc: Security, p: &str) -> Result<Connection, String> {
-    // Create a connection
-    // Get the connection
-    // Return the connection
-
-    let settings = ConnectionSettings {
-        id: "resin_io".to_string(),
-        uuid: "3c8e6e8b-b895-4b07-97a5-bbc192c3b436".to_string(),
-        ssid: "resin_io".to_string(),
-    };
+pub fn create(manager: &NetworkManager,
+              device: &Device,
+              access_point: &AccessPoint,
+              password: &str,
+              time_out: i32)
+              -> Result<Connection, String> {
+    let (path, _) = try!(manager.add_and_activate_connection(&device.path,
```

Comment:
This is the path of the active connection object, but we do not use it since it is filled in later in `get_connection`.